Description: Disable the intersphinx plugin
 This is to avoid network access during build.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2020-09-02

--- a/docs/build/conf.py	2020-09-02 13:39:55.857891290 +0200
+++ b/docs/build/conf.py	2020-09-02 13:40:22.081832983 +0200
@@ -34,7 +34,6 @@
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = [
     "sphinx.ext.autodoc",
-    "sphinx.ext.intersphinx",
     "changelog",
     "sphinx_paramlinks",
 ]
